spec/lib/analytics/campaign_edits_csv_builder_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require "#{Rails.root}/lib/analytics/campaign_edits_csv_builder"
+
+describe CampaignEditsCsvBuilder do
+  let(:campaign) { create(:campaign) }
+  let(:course) { create(:course) }
```

Comment:
This spec isn't being run with any edits. You can add a user to the course along with an edit or two, following the example of the CourseEditsCsvBuilder spec, so that it exercises the `row` method.

Reply:
Understood. I'll get to it.

Reply:
HI @ragesoss, I added a couple of tests as requested. I am having trouble adding more than one row but I did add one row and checked for content and total number of rows in CSV. Let me know if this is acceptable.